web-common/src/features/dashboards/time-series/utils.ts

```diff
@@ -134,17 +136,80 @@ export function getOrderedStartEnd(start: Date, stop: Date) {
 export function getFilterForComparedDimension(
   dimensionName: string,
   filters: V1Expression,
-  topListValues: string[],
 ) {
-  const includedValues = topListValues?.slice(0, 250);
-
   let updatedFilter = filterExpressions(
     filters,
     (e) => !matchExpressionByName(e, dimensionName),
   );
   if (!updatedFilter) {
     updatedFilter = createAndExpression([]);
   }
+  return updatedFilter;
+}
+
+export function transformAggregateDimensionData(
+  timeDimension: string,
+  dimensionName: string,
+  measures: string[],
+  dimensionValues: string[],
+  timeSeriesData: V1TimeSeriesValue[],
+  response: V1MetricsViewAggregationResponseDataItem[],
+) {
```

Comment:
Mind being more clear about what transformation this is? Maybe either add a return type, code comment, or a more descriptive function name.